components/VerifiableCredentials/Pause/index.tsx

```diff
@@ -0,0 +1,120 @@
+import styles from './styles.module.scss'
```

Comment:
how can we have the Pause & Unpause components implemented globally so they work for all xWallets? @devwistkey

Reply:
Sure, let me add this on notion @tralkan